share/availability/discovery/discovery.go

```diff
@@ -153,11 +155,11 @@ func (d *Discovery) handlePeerFound(ctx context.Context, peer peer.AddrInfo) boo
 	}
 
 	if !d.set.Add(peer.ID) {
-		log.Debugw("peer is already in discovery set", "peer", peer.ID)
+		log.Debug("peer is already in discovery set")
 		return false
 	}
 	d.onUpdatedPeers(peer.ID, true)
-	log.Debugw("added peer to set", "peer", peer.ID)
+	log.Debug("added peer to set")
```

Comment:
why did you remove logging the peer IDs?

Reply:
log has been shadowed by field log, that has peer field included, in the beginning of the func.
Reminds me one of the review comments to one of my PRs, when I did introduced field log, but didn't shadowed it, instead named it `logger` instead of common `log`

Reply:
Pls take a look at renamed `log` to `logger` version. I thin it should be less confusing